conbench/templates/run.html

```diff
@@ -46,59 +57,70 @@ <h3>Compare</h3>
   </div>
   <div class="mt-5">
     <h3>Results</h3>
-    <p>All benchmark results that have submitted for this CI run:</p>
-    <table class="table results-in-run-table table-hover table-borderless small">
-      <thead>
-        <tr>
-          <th scope="col">benchmark start time</th>
-          <th scope="col" style="width: 120px">benchmark name</th>
-          <th scope="col">case permutation</th>
-          <th scope="col" style="width: 55px">mean</th>
-          <th scope="col" style="width: 55px">error</th>
-        </tr>
-      </thead>
-      <tbody>
-        {% for benchmark in benchmarks %}
+    <p>{{  benchmarks | length }} benchmark result(s) submitted for this CI run:</p>
+    <div class="mt-5">
+      <table class="table table-hover table-borderless results-in-run-table small"
+             style="width:100%;
+                    display: none">
+        <thead>
           <tr>
-            <td style="white-space: nowrap;">{{ benchmark.display_timestamp }}</td>
-            <td>
-              <a href="{{ url_for('app.batch', batch_id=benchmark.batch_id) }}">{{ benchmark.display_bmname }}</a>
-            </td>
-            <td>
-              <a href="{{ url_for('app.benchmark-result', benchmark_result_id=benchmark.id) }}">{{
-              benchmark.display_case_perm }}</a>
-            </td>
-            <td style="white-space: nowrap;">{{ benchmark.display_mean }}</td>
-            <td>
-              {% if benchmark.error %}
-                <a href="{{ url_for('app.benchmark-result', benchmark_result_id=benchmark.id) }}">
-                  <i class="glyphicon glyphicon-exclamation-sign text-danger"
+            <th scope="col" style="width: 120px">benchmark name</th>
+            <th scope="col" style="width: 150px">start time (UTC)</th>
+            <th scope="col">result</th>
+            <th scope="col">case permutation</th>
+            <!--<th scope="col" style="width: 10%">measurements</th>-->
+            <th scope="col" style="width: 8%">measurement</th>
+            <th scope="col" style="width: 10%">
+              rel err
+              <sup><i class="bi bi-info-circle"
+   data-bs-toggle="tooltip"
+   data-bs-title=" Relative standard error: standard error of the mean in relationship to the mean value. Only built when at least three samples are reported by this result. ">
```

Comment:
Wonder if we should display the number of iterations here too?

Reply:
> Wonder if we should display the number of iterations here too?

yes, my goal was to make a table similar to this:
![image](https://github.com/conbench/conbench/assets/265630/45d38832-d79a-43d6-b3f3-2e0d63065914)

(rel url e.g. c-benchmarks/tpch/063dbebaf0534d68a462c372deb79a3c)

but then I noticed that we should first do https://github.com/conbench/conbench/issues/1341